utils/updateable_value.hh

```diff
@@ -210,4 +211,23 @@ updateable_value<T>::observe(std::function<void (const T&)> callback) const {
     return src ? src->observe(std::move(callback)) : dummy_observer<T>();
 }
 
+template <typename ValueType, typename UpdateableValueType>
+class simple_value_updater {
+    ValueType& _value;
+    utils::updateable_value<UpdateableValueType> _updateable_value;
+    serialized_action _updater;
+    utils::observer<UpdateableValueType> _observer;
+
+public:
+    simple_value_updater(ValueType& value, utils::updateable_value<UpdateableValueType> updateable_value, ValueType factor = 1)
```

Comment:
Why is the important name simple_value_updater used for multiplying?

I can see transforming_value_updater and unit_converting_value_updater.

Reply:
Changed to `transforming_value_updater` in v10 (using a std::function for the optional transform function)